dwitter/static/main.css

```diff
@@ -238,6 +238,8 @@ input.comment-submit:active {
 
 .author-remix-wrapper{
   padding: 10px 0;
+  display: inline-block;
```

Comment:
This works, and I don't think it will have any unintended quirks, although a more common way of resolving this issue is applying `overflow: hidden`, or using a `clearfix` hack (https://www.w3schools.com/howto/howto_css_clearfix.asp)